tests/views/timeline.py

```diff
@@ -67,14 +69,35 @@ def _get_suggestion_description(submission):
     display_name = (
         submission["suggestion__user__full_name"].strip()
         if submission["suggestion__user__full_name"].strip()
-        else submission["suggestion__user__username"])
-    author = format_html(u'<a href="{}">{}</a>', user_url, display_name)
+        else submission["suggestion__user__username"].strip())
+    params = {
+        'author': format_html(u'<a href="{}">{}</a>',
+                              user_url,
+                              display_name)
+    }
+    try:
+        comment = get_model().objects \
+                             .for_model(Suggestion) \
+                             .get(object_pk=submission["suggestion_id"])
+        params.update({
+            'comment': format_html(u'<span class="comment">{}</span>',
+                                   comment.comment),
+        })
+
+    except get_model().DoesNotExist:
```

Comment:
I find the overuse of `get_model()` annoying and misleading. Can't this be evaluated once at the top of the module, like `CommentModel = get_model()`?